tools/ci_build/build.py

```diff
@@ -199,6 +202,15 @@ def install_python_deps():
     dep_packages = ['setuptools', 'wheel', 'numpy']
     run_subprocess([sys.executable, '-m', 'pip', 'install', '--trusted-host', 'files.pythonhosted.org'] + dep_packages)
 
+def install_hosting_deps():
```

Comment:
This function should accept something like "`source_dir`" as parameter and use it as the first parameter in `os.path.join()`. Check other functions in this file as example.

Reply:
Done.